rq/queue.py

```diff
@@ -560,11 +566,13 @@ def enqueue_job(self, job, pipeline=None, at_front=False):
 
         job.origin = self.name
         job.enqueued_at = utcnow()
-
+        # TODO: Specify xx operator to prev'tent jobs from being double added to registry
```

Comment:
@alella  @selwin Should `xx` just be set to `ttl`?  I guess it would just update the score if it already exists depending on which other arguments we pass in.